gulpfile.js

```diff
@@ -75,7 +75,7 @@ const watchFiles = () => {
 
 const js = gulp.series(lint, scripts);
 const build = gulp.series(clean, gulp.parallel(styles, js));
-const watch = gulp.parallel(watchFiles, browserSync);
+const watch = gulp.series(build, gulp.parallel(watchFiles, browserSync));
```

Comment:
This change builds the project before watching for changes. This fixes an issue when pulling changes from repo, and then running the watcher without build, which caused some styling issues due to outdated CSS files